src/org/traccar/notification/NotificationFormatter.java

```diff
@@ -70,6 +70,23 @@ private NotificationFormatter() {
             + "Point: http://www.openstreetmap.org/?mlat=%3$f&mlon=%4$f#map=16/%3$f/%4$f%n"
             + "Time: %2$tc%n";
 
+    public static final String TITLE_TEMPLATE_TYPE_ALARM = "%1$s: alarm!";
+    public static final String MESSAGE_TEMPLATE_TYPE_ALARM = "Device: %1$s%n"
+            + "Alarm: %5$s%n"
+            + "Point: http://www.openstreetmap.org/?mlat=%3$f&mlon=%4$f#map=16/%3$f/%4$f%n"
```

Comment:
I like it. May be it can be special API request that load report with one position or something like that.